src/clic.adoc

```diff
@@ -1624,7 +1629,7 @@ the exception handler privilege mode contains a trap handler function address in
 
 The overall effect is:
 
-     pc := M[VTBASE + XLEN/8 * exccode] & ~1
+     pc := M[VTBASE + XLEN/8 * exccode] & MASK
```

Comment:
This is redundant with the definition above. No new information here, but pretty much just a copy.

Reply:
I didn't add the "pc := ..." text, I just corrected what was already there since I'm trying to minimize changes. If you think this should be removed (and not corrected) maybe you can file another GitHub issue and make the change.